client/catapult/tests/catapult/utils/CatapultExceptionTests.cpp

```diff
@@ -77,11 +77,35 @@ namespace catapult {
 #define PTRSUFFIX "*"
 #endif
 
+		// Boost function name got updated and now return in the form below.  This function coverts the function name to the
+		// expected value in the exception.
+		// void catapult::CatapultExceptionTests_CanCopyConstructException() [with TTraits = catapult::{anonymous}::RuntimeErrorTraits]
+		std::string convertToExceptionFunctionName(const std::string& functionFullName) {
```

Comment:
why this adaptation instead of updating expected FunctionName passed in ?

Reply:
yea, probably make more sense to update ``functionFullName`` thus dont have to return another string.
I was thinking for easy of calling the function on the same line as the assignment. :)

Reply:
hmm...it seem the expected FunctionName is ``const string``.  I guess I could remove it but not sure if this make sense.